prime-router/src/main/kotlin/azure/BlobAccess.kt

```diff
@@ -30,10 +29,6 @@ class BlobAccess {
         return Pair(getBodyFormat(report).toString(), outputStream.toByteArray())
     }
 
-    private fun createBodyFilename(report: Report): String {
```

Comment:
This code was pretty redundant with filename creation code in Report.kt.   Just removed it, as the report.name is already in the form of a filename.